examples/slash_commands/option_choice.py

```diff
@@ -0,0 +1,43 @@
+from disnake import Option, OptionChoice, Interaction, Intents
+from disnake.ext import commands
+from disnake.ext.commands import slash_command
+
+import random
+
+intents = Intents.all()
+# In production please choose the intents coresponding to your use case.
+
+# We create the instance of the bot.
+bot = commands.Bot(intents=intents)
+
+# For this example we're going to make a rock-paper-scissors game
+# which will use OptionChoice as the input.
+@slash_command(name='rps', description='Play a game of rock-paper-scissors with the bot.',
```

Comment:
This decorator is the best for cogs, but it's not very convenient outside cogs, because you'll have to call `bot.add_slash_command(rps)`. Instead of using `@commands.slash_command` here, use `@bot.slash_command`.